### PR TITLE
FEAT: OmniSciDB - Add support to week extraction

### PR DESCRIPTION
**Pull request by xmnlab**
OmniSciDB - Add support to week extraction

ref: #1499

### CODE REVIEW

**Comment by xmnlab** [comment]
this PR is ready for review! thanks!

File: ibis/omniscidb/operations.py [hunk 1]
```diff
@@ -286,6 +289,30 @@ def extract_field_formatter(translator, expr, sql_attr=sql_attr):
     return extract_field_formatter
 
 
+def _extract_field_dow_name(sql_attr):
+    def extract_field_formatter(translator, expr, sql_attr=sql_attr):
```
Comment (jreback) [inline ibis/omniscidb/operations.py:293]: did you enable tests for this?

Comment (xmnlab) [reply ibis/omniscidb/operations.py:293]: as it is not unsupported anymore the current tests should check it https://github.com/ibis-project/ibis/blob/master/ibis/tests/all/test_temporal.py#L361-L397

**Comment by jreback** [comment]
lgtm.

@datapythonista see if you can merge here.

**Comment by datapythonista** [comment]
No, same messages as in the screenshot I shared:

```
Merging is blocked
The base branch restricts merging to authorized users. Learn more about protected branches.

You’re not authorized to merge this pull request.
```

**Comment by jreback** [comment]
try again @datapythonista 
confusingly github has a *separate* list for branch protection (which i think overrides the 'write' team access).

**Comment by datapythonista** [comment]
Yep, all good now, thanks @jreback 

I was a bit confused, because I'm not in this list: https://github.com/orgs/ibis-project/people

But apparently is not needed.

Thanks @xmnlab for the PR

**Comment by jreback** [comment]
yeah i am not sure the teams are setup correctly, will fix at some point.

**Pull request merged**